powerhome/playbook
# converted dashboard value to typescript

#### Screens

<img width="321" alt="Screen Shot 2023-02-15 at 11 49 56 AM" src="https://user-images.githubusercontent.com/73671109/219097015-26265ab1-2023-4fe6-98f1-33084cebdd53.png">

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-221)

#### How to test this

Use dashboard value kit and ensure it works as expected after typescript conversion

#### Checklist:
